client/components/data-labels/labels.directive.js

```diff
@@ -60,10 +63,36 @@ angular.module('probedock.dataLabels').directive('simpleLabel', function() {
   $scope.getTypeClass = function() {
     return $scope.type ? 'label-' + $scope.type : '';
   };
+})
+.controller('ProjectVersionLabelCtrl', function($scope, projectNameFilter) {
+  if (!$scope.labelSize) {
+    $scope.labelSize = 30;
+  }
+
+  $scope.getTooltip = function () {
+    return $scope.versionOnly ? $scope.projectVersion : projectNameFilter($scope.project) + ' ' + $scope.projectVersion;
+  };
+
+  $scope.tooltipEnabled = function () {
+    return !$scope.truncate && $scope.getTooltip().length > $scope.labelSize;
+  };
+
+  $scope.getLabel = function () {
+    var str = $scope.getTooltip();
+
+    if (!$scope.truncate && str.length > $scope.labelSize) {
```

Comment:
I don't understand this condition. Why do we truncate if the `truncate` argument is false?